src/main/resources/db/changelog/changes/v1.1.0/update-drv-item-details-definition.xml

```diff
@@ -248,6 +248,10 @@
                 "valueJsonPath": "$.mtypes.*.id",
                 "labelJsonPath": "$.mtypes.*.name"
               },
+              "source": {
+                "columnName": "item_material_type",
+                "entityTypeId": "0cb79a4c-f7eb-4941-a104-745224ae0292"
+              },
```

Comment:
The query builder uses the property to configure how it requests the available values, so we actually still need it even if the values aren't coming from the DB. In this case, I set it up to just ask for the values for this column for the Items entity type, but an alternative approach could be to use the existing material types entity type and modify that one so that the values come from an API instead of the DB. I opted to do it all in this entity type because that will allow us to kill off an entity type which just feels good